cypress/integration/configuration/productTypes/purchaseWithProductTypes.js

```diff
@@ -0,0 +1,219 @@
+import faker from "faker";
+
+import { createAttribute } from "../../../apiRequests/Attribute";
+import { createCategory } from "../../../apiRequests/Category";
+import {
+  checkoutShippingAddressUpdate,
+  checkoutShippingMethodUpdate,
+  checkoutVariantsUpdate,
+  completeCheckout,
+  createCheckout
+} from "../../../apiRequests/Checkout";
+import { getOrder } from "../../../apiRequests/Order";
+import { createTypeProduct } from "../../../apiRequests/Product";
+import { getDefaultChannel } from "../../../utils/channelsUtils";
+import {
+  addPayment,
+  createAndCompleteCheckoutWithoutShipping,
+  createWaitingForCaptureOrder
+} from "../../../utils/ordersUtils";
+import {
+  createProductInChannel,
+  deleteProductsStartsWith
+} from "../../../utils/products/productsUtils";
+import {
+  createShipping,
+  deleteShippingStartsWith
+} from "../../../utils/shippingUtils";
+
+describe("Purchase products with all products types", () => {
+  const startsWith = `CyPurchaseByType`;
+  const name = `${startsWith}${faker.datatype.number()}`;
+  const email = `${startsWith}@example.com`;
+
+  let defaultChannel;
+  let address;
+  let warehouse;
+  let attribute;
+  let category;
+  let shippingMethod;
+  let createProductData;
+
+  before(() => {
+    cy.clearSessionData().loginUserViaRequest();
+    deleteProductsStartsWith(startsWith);
```

Comment:
Is this lack of indentation is intentional?

Reply:
Yes, there is no need for indentation if line before ended with ;